src/protagonist/API/Features/Image/Validation/HydraImageValidator.cs

```diff
@@ -36,7 +36,11 @@ public HydraImageValidator(IOptions<ApiSettings> apiSettings)
             .When(_ => !apiSettings.Value.DeliveryChannelsEnabled)
             .WithMessage("Delivery channels are disabled");
         
-        RuleForEach(a => a.DeliveryChannels)
+        RuleFor(a => a.ModelId).Must(id => id.IndexOfAny(apiSettings.Value.RestrictedAssetIdCharacters.ToCharArray()) == -1)
+            .When(asset => !asset.ModelId.IsNullOrEmpty())
+            .WithMessage("Asset id contains a restricted character");

```

Comment:
Restricted characters should be included in the exception message to help the caller resolve the issue